tests/test_reimburse.py

```diff
@@ -0,0 +1,131 @@
+import json
+
+from pymongo import MongoClient
+import pytest
+import requests
+
+import authorize
+import config
+
+from testing_utils import *
+import util
+import validate
+import reimburse
+
+email = "director77@gmail.com"
+pword = "1234566"
+token = ""
+old_col = ""
+
+users_result = [20, 0, 0, 60, 60, 40, 59.99]
+users = [
+	    {
+	    	"travelling_from" : {
+		    "is_real" : True,
+		    "mode" : "car",
+		    "addr_ready" : True,
+		    "formatted_addr" : "Millburn, NJ, USA"
+		}
+	    },
+	    {   
+		"travelling_from" : {
+		    "is_real" : False,
+		    "mode" : None,
+		    "addr_ready" : False,
+		    "formatted_addr" : None
+		}
+	    },
+	    {
+		"travelling_from" : {
+		    "is_real" : True,
+		    "mode" : "car",
+		    "addr_ready" : True,
+		    "formatted_addr" : "Princeton, NJ, USA"
+		}
+	    },
+	    {
+		"travelling_from" : {
+		    "is_real" : True,
+		    "mode" : "bus",
+		    "addr_ready" : True,
+		    "formatted_addr" : "College Park, MD, USA"
+		}
+	    },
+	    {
+		"travelling_from" : {
+		    "is_real" : True,
+		    "mode" : "train",
+		    "addr_ready" : True,
+		    "formatted_addr" : "Providence, RI, USA"
+		}
+	    },
+	    {
+		"travelling_from" : {
+		    "is_real" : True,
+		    "mode" : "car",
+		    "addr_ready" : True,
+		    "formatted_addr" : "White Plains, NY, USA"
+		}
+	    },
+	    {
+		"travelling_from" : {
+		    "is_real" : True,
+		    "mode" : "car",
+		    "addr_ready" : True,
+		    "formatted_addr" : "Pittsburgh, PA, USA"
+		}
+	    }
+        ]
+
+
+def payload(auth_email=email):
+    return {"email" : email,
+            "token" : token}
+
+def setup_module(m):
+    db = util.get_db()
+    db["test-users"].drop()
```

Comment:
maybe we should make a ticket for decorators for common setup/teardown stuff. like making a user or switching to a test database